src/codegen.rs

```diff
@@ -150,9 +150,22 @@ impl<'ctx> CodeGen<'ctx> {
 
         // TODO: This is how the Kaleidoscope tutorial does it, but it feels kinda yucky to mutate state like this..?
         self.named_values.clear();
+
         for param in the_fn.get_param_iter() {
+            let param_as_float: FloatValue = if param.is_float_value() {
```

Comment:
I think Inkwell should expose a `get_name()` fn for any type wrapping `LLVMValue`..